nuncio/style-nuncio.css

```diff
@@ -18,6 +18,83 @@ html, body {
   border-radius: 10px;
 }
 
+.filled-assessments{
+  position: absolute;
+  width: 817px;
+  height: 427px;
+  left: 36px;
+  top: 258px;
```

Comment:
Remove the position: absolute, and the left and top properties here. We won't be needing them when we place everything!